app/src/main/java/cloudcity/networking/models/Iperf3NetworkDataModel.java

```diff
@@ -0,0 +1,68 @@
+package cloudcity.networking.models;
+
+import androidx.annotation.NonNull;
+
+import com.google.gson.annotations.SerializedName;
+
+import cloudcity.dataholders.MetricsPOJO;
+import de.fraunhofer.fokus.OpenMobileNetworkToolkit.DataProvider.LocationInformation;
+
+public class Iperf3NetworkDataModel extends NetworkDataModel {
+
+    @SerializedName("category")
+    final private String category = "Iperf3";
+
+    @SerializedName("accuracy")
+    private double accuracy;
+    @SerializedName("speed")
+    private double speed;
+
+    public Iperf3NetworkDataModel(
+            @NonNull MetricsPOJO.UploadMetrics upload,
+            @NonNull MetricsPOJO.DownloadMetrics download,
+            @NonNull LocationInformation location
+    ) {
+        super(location.getLatitude(), location.getLongitude(), new Iperf3ValuesModel(upload, download));
+        this.accuracy = location.getAccuracy();
+        this.speed = location.getSpeed();
+    }
```

Comment:
nah man, what's hte point of validating something outside of my control anyways? what if we're moving in reverse, wouldn't the speed be negative then?

Reply:
well, i don't know. but they most certainly won't cause as much harm as having to wrap each instantiation of Iperf3NetworkDataModel in a try/catch in case it throws would. we're not gaining anything with this change, only tying our hands behind our backs a bit more.

resolving.